Cargo.toml

```diff
@@ -9,9 +9,17 @@ authors = [
 license = "MIT"
 description = "Raw unsafe system binding for openvr"
 repository = "https://github.com/rust-openvr/rust-openvr-sys"
-
+exclude = [
+    "openvr/samples/*",
+    "openvr/bin/*",
+    "openvr/lib/*",
```

Comment:
Excluding the libs might cause issues when someone downloads rust-openvr-sys from cargo I guess.